tests/accessibility/getAccessibilityTree.test.ts

```diff
@@ -0,0 +1,126 @@
+import type { ElementHandle } from 'pleasantest';
+import {
+  experimentalGetAccessibilityTree as getAccessibilityTree,
+  withBrowser,
+} from 'pleasantest';
+
+test(
+  'basic use cases',
+  withBrowser(async ({ utils, page }) => {
+    await utils.injectHTML(`
+      <main>
+        <button>
+          <span aria-hidden="true">+</span>
+          <span>Add to cart</span>
+        </button>
+        <h1>hiiii</h1>
+        <div role="button" tabindex="-1">foo &gt bar</div>
+      </main>
+    `);
+    const body: ElementHandle = await page.evaluateHandle(() => document.body);
+    expect(await getAccessibilityTree(body)).toMatchInlineSnapshot(`
+      main
+        button "Add to cart"
+        heading "hiiii"
+        button "foo > bar"
+    `);
+    await utils.injectHTML(`
+      <ul>
+        <li>something</li>
+        <li>something else</li>
+      </ul>
+    `);
+    expect(await getAccessibilityTree(body)).toMatchInlineSnapshot(`
+      list
+        listitem
+        listitem
+    `);
+    expect(await getAccessibilityTree(body, { includeText: true }))
+      .toMatchInlineSnapshot(`
+        list
+          listitem
+            text "something"
+          listitem
+            text "something else"
+      `);
+    await utils.injectHTML(`
+      <button aria-describedby="click-me-description">click me</button>
+      <div id="click-me-description">extended description</div>
+    `);
+    expect(await getAccessibilityTree(body)).toMatchInlineSnapshot(`
+      button "click me"
+        ↳ description: "extended description"
+    `);
+    expect(await getAccessibilityTree(body, { includeText: true }))
+      .toMatchInlineSnapshot(`
+        button "click me"
+          ↳ description: "extended description"
+          text "click me"
```

Comment:
I wonder if this should be included when `includeText` is true since the text is already displayed in the accessible name? I'm not sure what the right answer is. Curious what you and others think.

Reply:
I thin you probably have it right as-is. Things could get confusing quickly if you start omitting text that's used as a name/description.

Reply:
Good question! Ideally I would like it not to be shown. But `dom-accessibility-api` doesn't yet have a way to see which nodes it already used when computing the name/description: https://github.com/eps1lon/dom-accessibility-api/issues/728. I'll open a PR there so we can have that feature.
Also, FWIW chrome's accessibility tree shows the text nodes in cases like this, even if they are already used in computing the name/description.

Reply:
Yeah that makes sense. I think Chrome might be doing it right. I'd rather be verbose than remove things in a way that could be confusing (and I could see this becoming confusing with `aria-labelledby` in the mix.)

Reply:
I think there are some cases where the text is "used up" in computing the name, and the screen reader can't read the text separately from the name. Ideally in those cases we wouldn't show `text`. I'll think about it more to see if there is a consistent way to make this work. (outside of this PR)

Reply:
Gotcha, that makes sense :+1: